event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/JdbcEventStreamRepository.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.services.eventsourcing.repository.jdbc;
+
+import uk.gov.justice.services.eventsourcing.repository.jdbc.eventstream.EventStreamJdbcRepository;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.eventstream.EventStreamObject;
+
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import javax.inject.Inject;
+
+/**
+ */
+public class JdbcEventStreamRepository implements EventStreamRepository {
```

Comment:
So this is exactly the same as the Injected repository class but with different names.

Should this be called JDBC ?

Reply:
Right now doing nothing, thinking of removing will speak to David to confirm